src/procedures/migrate-zigbee-zha-only.html

```diff
@@ -0,0 +1,11 @@
+---
+layout: getting-started
+description: Home Assistant SkyConnect
+guideName: Migrate a ZHA network
+
+---
+<div class="container">
+    <p class="text-center note">Follow this guide if you have a Zigbee Home Assistant (ZHA) network running and want to migrate to SkyConnect.</p>
+    <p class="text-center note">Note: this procedure only helps migrate basic network settings to your Home Assistant SkyConnect, customizations like device names will be lost and automations will need to be updated. Most powered devices like lightbulbs will be re-discovered over time (you can speed this up by rebooting the device) but your battery-powered devices may need to be re-joined to the network for the migration to complete. There is currently no migration path to migrate all settings and devices.</p>
```

Comment:
In this case, this is not really true, as we retain the ZHA database with all it's devices etc. I think we can pretty much drop that sentence.

Not sure if there are other caveats @puddly ?

Reply:
In the case of a ZHA migration, yes, there is no loss of any customization: everything will work exactly as before.